src/m_misc.c

```diff
@@ -2886,12 +2886,7 @@ void M_SaveDefaults (void)
                 fprintf(f, "%c", c);
               }
               else
-              {
-                const char *s = M_GetNameForKey(v->value);
-                if (!s)
-                  s = "JUNK";
-                fprintf(f, "%s", s);
-              }
+                fprintf(f, "%s", M_GetNameForKey(v->value));
```

Comment:
How about just don't `fprintf` anything `if (s == NULL)`?

Reply:
In that case, if you have already assigned a different key or button to the same action, it will still print the comma (in the config file) or the plus (in the menu), but nothing after that.

Reply:
"JUNK" means, there is some key assigned, but I don't have a name for it, so don't expect this binding to stay persistent across restarts. We could also call it "UNKNOWN" or "N/A" or just "?" if you prefer.

Reply:
> it will still print the comma (in the config file)

I think it's OK.

> or the plus (in the menu), but nothing after that.

We can keep `"JUNK"` in the menu.